src/components/Task/Modal/Form/Complete/TaskModalCompleteForm.test.tsx

```diff
@@ -283,261 +579,5 @@ describe('TaskModalCompleteForm', () => {
-      'None',
-      'Call',
-      'Email',
-      'Text Message',
-      'Facebook Message',
-      'Talk To In Person',
-      'Appointment',
-      'Prayer Request',
-      'Thank',
-    ]);
-  });
-
-  it('has correct options for PRE_CALL_LETTER', () => {
-    const { results, nextActions } = getOptions(ActivityTypeEnum.PreCallLetter);
-    expect(results).toEqual(['Completed', 'Received']);
-    expect(nextActions).toEqual([
-      'None',
-      'Call',
-      'Email',
-      'Text Message',
-      'Facebook Message',
-      'Talk To In Person',
-    ]);
-  });
-
-  it('has correct options for REMINDER_LETTER', () => {
-    const { results, nextActions } = getOptions(
-      ActivityTypeEnum.ReminderLetter,
-    );
-    expect(results).toEqual(['Completed', 'Received']);
-    expect(nextActions).toEqual([
-      'None',
-      'Call',
-      'Email',
-      'Text Message',
-      'Facebook Message',
-      'Talk To In Person',
-    ]);
-  });
-
-  it('has correct options for SUPPORT_LETTER', () => {
-    const { results, nextActions } = getOptions(ActivityTypeEnum.SupportLetter);
-    expect(results).toEqual(['Completed', 'Received']);
-    expect(nextActions).toEqual([
-      'None',
-      'Call',
-      'Email',
-      'Text Message',
-      'Facebook Message',
-      'Talk To In Person',
-    ]);
-  });
-
-  it('has correct options for TALK_TO_IN_PERSON', () => {
-    const { results, nextActions } = getOptions(
-      ActivityTypeEnum.TalkToInPerson,
-    );
-    expect(results).toEqual(['Completed', 'Received']);
-    expect(nextActions).toEqual([
-      'None',
-      'Call',
-      'Email',
-      'Text Message',
-      'Facebook Message',
-      'Talk To In Person',
-      'Appointment',
-      'Prayer Request',
-      'Thank',
-    ]);
-  });
-
-  it('has correct options for TEXT_MESSAGE', () => {
-    const { results, nextActions } = getOptions(ActivityTypeEnum.TextMessage);
-    expect(results).toEqual(['Completed', 'Received']);
-    expect(nextActions).toEqual([
-      'None',
-      'Call',
-      'Email',
-      'Text Message',
-      'Facebook Message',
-      'Talk To In Person',
-      'Appointment',
-      'Prayer Request',
-      'Thank',
-    ]);
-  });
-
-  it('has correct options for THANK', () => {
-    const { results, nextActions } = getOptions(ActivityTypeEnum.Thank);
-    expect(results).toEqual(['Completed', 'Received']);
-    expect(nextActions).toEqual([
-      'None',
-      'Call',
-      'Email',
-      'Text Message',
-      'Facebook Message',
-      'Talk To In Person',
-    ]);
-  });
-
-  it('has correct options for TO_DO', () => {
-    const { results, nextActions } = getOptions(ActivityTypeEnum.ToDo);
-    expect(results).toEqual([]);
-    expect(nextActions).toEqual([]);
-  });
-
-  it('has correct options for null', () => {
-    const { results, nextActions } = getOptions(undefined);
-    expect(results).toEqual([]);
-    expect(nextActions).toEqual([]);
-  });
+  taskModalTests(Components);
```

Comment:
This is because there are duplicate tests in multiple files. To avoid duplication, I created a function which would run the same tests in multiple files.

Reply:
I only see `taskModalTests` used here though, not in multiple places.

Reply:
I think I had planned to add the Log and Task modals to these tests, but I think it was unfinished. It would be good to add these tests to both of those modals, but it will need some refactoring